Classes/Command/JobCommandController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace Netlogix\Aws\ElasticBeanstalk\Command;
+
+/*
+ * This file is part of the Netlogix.Aws.ElasticBeanstalk package.
+ */
+
+use Flowpack\JobQueue\Common\Job\JobInterface;
+use Flowpack\JobQueue\Common\Job\JobManager;
+use TYPO3\Flow\Annotations as Flow;
+
+/**
+ * @Flow\Scope("singleton")
+ */
+class JobCommandController extends \TYPO3\Flow\Cli\CommandController
+{
+
+	/**
+	 * @Flow\Inject
+	 * @var JobManager
+	 */
+	protected $jobManager;
+
+	/**
+	 * Creates and schedules a new Job
+	 *
+	 * @param string $queue the queue to use for scheduling
+	 * @param string $job the class name of the job to schedule
+	 * @param string $arguments the constructor arguments of the job to schedule (as json string)
+	 */
+	public function scheduleCommand($queue, $job, $arguments = null)
+	{
+		if ($arguments === null && count($this->request->getExceedingArguments()) > 0) {
+			$arguments = join('', $this->request->getExceedingArguments());
+		}
+		$arguments = $arguments === null ? [] : json_decode($arguments, true);
+
+		if (count($arguments) === 0) {
+			$job = new $job();
```

Comment:
That code is actually no longer in this PR. I moved all of this stuff to a separate package (and I use different names there).